docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -206,7 +206,7 @@ Use the platform default configuration for all settings that affect screenshots,
 
 ### Include window shadow and rounded corners
 
-Screenshots must include the native toolkit shadow and rounding. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
+Screenshots must include the native toolkit's (or system's if the platform renders titlebars) titlebar, shadow and rounding. App stores do not add a shadow after the fact and without one screenshots can look glitchy or have low contrast depending on the background. Apps that are always fullscreen (such as most games) are exempt from this, but in these cases all screenshots must be fullscreen. Do not use a mix of windowed and fullscreen screenshots.
```

Comment:
can we somehow not say titlebar twice? :D

Reply:
And the `'s` feel weird IMO

Reply:
There, feel that should be a good enough solution given all the feedback until now.